gtest/src/wasm_executor.rs

```diff
@@ -19,42 +19,36 @@
 use core_processor::{Ext, ProcessorContext, ProcessorExt};
 use gear_backend_common::TerminationReason;
 use gear_backend_wasmi::{
-    env::{DefinedHostFunctions, EnvironmentDefinitionBuilder, GuestExternals, Runtime},
+    env::{EnvironmentDefinitionBuilder, GuestExternals},
     funcs::{FuncError, FuncsHandler as Funcs},
+    runtime::Runtime,
     MemoryWrap,
 };
 use gear_core::{
-    env::{Ext as ExtTrait, ExtCarrier},
+    env::Ext as ExtTrait,
     gas::{GasAllowanceCounter, GasCounter, ValueCounter},
     memory::{AllocationsContext, PageBuf, PageNumber, WasmPageNumber},
     message::{IncomingMessage, MessageContext, Payload},
     program::Program,
 };
 use std::{collections::BTreeMap, mem};
-use wasmi::{
-    memory_units::Pages, MemoryInstance, MemoryRef, ModuleInstance, ModuleRef, RuntimeValue,
-};
+use wasmi::{memory_units::Pages, MemoryInstance, MemoryRef, ModuleInstance, RuntimeValue};
 
 use crate::{manager::ExtManager, Result, TestError, MAILBOX_THRESHOLD};
 
 /// Binary meta-functions executor for testing purposes
-pub(crate) struct WasmExecutor {
-    instance: ModuleRef,
-    store: Runtime<Ext>,
-    memory: MemoryRef,
-    defined_host_functions: DefinedHostFunctions<Runtime<Ext>, <Ext as ExtTrait>::Error>,
-}
+pub(crate) struct WasmExecutor;
```

Comment:
Do we ever need this struct after all of these changes?

Reply:
We need to pass struct like this https://github.com/gear-tech/gear/blob/e4ac4d069ae1b152f85d641572fe09788a6e2721/pallets/gear/src/lib.rs#L791